test/e2e/util/util.go

```diff
@@ -124,10 +126,20 @@ func WaitForPodToBeReady(client *kubernetes.Clientset, waitSeconds time.Duration
 	}
 }
 
-func PodWatcher(client *kubernetes.Clientset, clusterID, namespace, podName string, podReady, stopCh chan bool) {
-	watcher, err := client.CoreV1().Pods(namespace).Watch(context.TODO(), metav1.ListOptions{
-		FieldSelector: fields.OneTermEqualSelector(api.ObjectNameField, podName).String(),
-	})
+func PodWatcher(location string, client *kubernetes.Clientset, clusterID, namespace, podName string, podReady, stopCh chan bool) {
+	var watcher watch.Interface
+	var err error
+
+	if location == "home" {
```

Comment:
I think that a comment there is needed to understand properly the different test-cases.